content/300-guides/500-other/900-advanced-database-tasks/06-typesafe-raw-sql/01-gis-support.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Using SafeQL for unsupported features'
+metaTitle: 'Using SafeQL for unsupported features (Guide)'
```

Comment:
Isn't the content specific to GIS? Or is that just supposed to be an example and this is a more general description?

Reply:
The content is specific to GIS but is meant to be generally applicable. See https://github.com/prisma/docs/pull/5419/files#diff-88c367bfdad7e340b81014f51a54dbbbaf5387e7846ea059b6f32d6ba0a0c663R11

Reply:
Then I would suggest to restructure the article:

- Rename file to `safeql-gis.mdx` or similar
- Start with explanation of what SafeQL is and how it works
- Explain state of GIS in Prisma
- Then example that uses SafeQL in extension to make usage of GIS nicer